### PR TITLE
Bug/jenkins 37603 favorite hover state

### PR DESCRIPTION
**Pull request by cliffmeyers**
**Description**
- See [JENKINS-37603](https://issues.jenkins-ci.org/browse/JENKINS-37603).

**Submitter checklist**
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests

**Reviewer checklist**
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -9,4 +11,4 @@
 - [ ] Reviewed the code
 - [ ] Verified that the appropriate tests have been written or valid explanation given
 
-@reviewbybees 
+@reviewbybees
```
Comment (scherler) [inline PULL_REQUEST_TEMPLATE]: @jenkinsci/code-reviewers as well?

File: src/js/components/favorite/Favorite.jsx [hunk 2]
```diff
@@ -48,13 +51,13 @@ export class Favorite extends Component {
...
...5A623"/><path d="M192 0h24v24h-24V0z"/><g stroke="#F5A623" stroke-linecap="square"><path d="M201 8l-1.5-1.5M207 8l1.5-1.5M199.051 14.054l-1.576 1.42M210.46 15.038L209 14M203.495 18.849l.01-1.12"/></g></g><g><path d="M216 0h24v24h-24V0z"/><path d="M228 18.324l7.416 4.476-1.968-8.436L240 8.688l-8.628-.732L228 0l-3.372 7.956-8.628.732 6.552 5.676-1.968 8.436L228 18.324z" fill="#F5A623"/><path d="M216 0h24v24h-24V0z"/></g><g><path d="M264 0h24v24h-24V0z"/><path className="star-filled" d="M276 17.27l6.18 3.73-1.64-7.03L286 9.24l-7.19-.61L276 2l-2.81 6.63-7.19.61 5.46 4.73-1.64 7.03 6.18-3.73z" fill="#F5A623"/><path d="M264 0h24v24h-24V0z"/></g><g><path d="M240 0h24v24h-24V0z"/><path d="M252 17.797l6.798 4.103-1.804-7.733L263 8.964l-7.909-.671L252 1l-3.091 7.293-7.909.671 6.006 5.203-1.804 7.733L252 17.797z" fill="#F5A623"/><path d="M240 0h24v24h-24V0z"/></g><circle stroke="#F5A623" stroke-width=".5" cx="156" cy="12" r="8"/><circle stroke="#F5A623" stroke-width=".3" cx="180" cy="12" r="10"/><circle stroke="#F5A623" stroke-width=".1" cx="204" cy="12" r="11"/><circle stroke="#F5A623" stroke-width=".8" cx="132" cy="12" r="7"/><circle stroke="#F5A623" cx="108.5" cy="12.5" r="5.5"/></g></svg>
```
Comment (scherler) [inline src/js/components/favorite/Favorite.jsx]: one big long line, really?

**Comment by scherler** [comment]
LGTM

File: src/js/components/favorite/Favorite.jsx (hunk 2 above)
Comment (michaelneale) [inline src/js/components/favorite/Favorite.jsx]: it would be a lot of lines otherwise... is there a better way to do this?

Comment (cliffmeyers) [inline src/js/components/favorite/Favorite.jsx]: Perhaps an approach like this: https://github.com/matthewwithanm/react-inlinesvg   Not sure I want to use the lib unless it's very small and has no deps. Could roll our own...

Comment (cliffmeyers) [inline src/js/components/favorite/Favorite.jsx]: And I don't mind properly formatting the SVG in the file too. In retrospect it was silly to paste it all on one line. :/

Comment (cliffmeyers) [inline src/js/components/favorite/Favorite.jsx]: Formatted in 25ded95

**Pull request merged**